clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -92,4 +94,27 @@ public void testSeekNegative() {
         consumer.assign(Arrays.asList(new TopicPartition("nonExistTopic", 0)));
         consumer.seek(new TopicPartition("nonExistTopic", 0), -1);
     }
+
+    @Test
+    public void testInterceptorConstructorClose() throws Exception {
+        Properties props = new Properties();
+        // test with client ID assigned by KafkaConsumer
+        props.setProperty(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, "localhost:9999");
+        props.setProperty(ConsumerConfig.INTERCEPTOR_CLASSES_CONFIG, MockConsumerInterceptor.class.getName());
+
+        final int oldInitCount = MockConsumerInterceptor.INIT_COUNT.get();
+        final int oldCloseCount = MockConsumerInterceptor.CLOSE_COUNT.get();
+
+        KafkaConsumer<String, String> consumer = new KafkaConsumer<String, String>(
+                props, new StringDeserializer(), new StringDeserializer());
+        Assert.assertEquals(oldInitCount + 1, MockConsumerInterceptor.INIT_COUNT.get());
+        Assert.assertEquals(oldCloseCount, MockConsumerInterceptor.CLOSE_COUNT.get());
+
+        consumer.close();
+        Assert.assertEquals(oldInitCount + 1, MockConsumerInterceptor.INIT_COUNT.get());
+        Assert.assertEquals(oldCloseCount + 1, MockConsumerInterceptor.CLOSE_COUNT.get());
```

Comment:
Can we simplify these asserts to just use `1` now that we reset the counters after each test?